test/app-luatest/gh_11492_popen_is_closed_test.lua

```diff
@@ -0,0 +1,42 @@
+local popen = require('popen')
+
+local t = require('luatest')
+local g = t.group()
+
+g.test_popen_is_closed = function()
+    local ph = popen.new({'true'})
```

Comment:
Sadly, `popen` doesn't perform search in `PATH`, see #5044.

Use `/bin/true` (a path to the executable file) or `popen.shell` (to let the shell perform the `PATH` lookup).